decidim-proposals/lib/decidim/proposals/engine.rb

```diff
@@ -88,6 +94,14 @@ class Engine < ::Rails::Engine
         end
       end
 
+      initializer "decidim_proposals_mentioned_listener" do
+        Decidim::Comments::CommentCreation.subscribe do |data|
+          comment = Decidim::Comments::Comment.find(data[:comment_id])
+          metadata = data[:metadatas][:proposals]
+          Decidim::Proposals::NotifyProposalsMentionedJob.perform_later(comment, metadata)
```

Comment:
As commented previously, it would be better just to pass the comment id and proposal ids to avoid serializing all that content.

Reply:
Done